mechanics/RELEASE-SCHEDULE.md

```diff
@@ -5,25 +5,33 @@ weight: 50
 type: "docs"
 ---
 
-Knative releases every 6 weeks. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
+Knative releases quarterly. As much as possible, releases should be driven by automation, and repos should be ready to release at any point. It should also be possible to produce and consume nightly artifacts.
 
-With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 6+ months of releases.
+With that said, it can be useful to have a list of when future releases will happen, so this document provides a schedule for the next 12 months of releases.
+
+## Upcoming releases
 
 | Release | Date       | EOL        | Min K8s Version | Notes                         |
 | ------- | ---------- | ---------- | --------------- | ----------------------------- |
-| 1.8     | 2022-10-04 | 2023-04-04 | 1.23            | |
-| 1.9     | 2022-11-15 | 2023-05-16 | 1.23            | |
-| 1.10    | 2023-01-10 | 2023-06-27 | 1.24            | |
-
+| 1.8     | 2022-10-18 | 2023-05-02 | 1.23            | |
```

Comment:
Shouldn't this date be 04-25?

Reply:
Previously we had one week overlap. I think it's a good idea to keep it. The EOL was calculated by taking n+2 release date and adding one week to it. That way you ensure the n+2 release be already out, when n hits his EOL.